docs/develop/installation.rst

```diff
@@ -16,23 +16,25 @@ It is installed from **pip** on all platforms:
 
 **Installation notes:**
 
-* Mac and Linux require you prefix the command with ``sudo``.
+* Install `dronekit` with `pip` inside a virtualenv:
     
   .. code-block:: bash
 
-      sudo pip install dronekit
+      python3 -m venv .venv
+      . .venv/bin/activate
+      pip install dronekit
       
 * On Linux you may need to first install **pip** and **python-dev**:
     
   .. code-block:: bash
 
       sudo apt-get install python-pip python-dev
       
-  Alternatively, the *easy_install* tool provides another way to install pip:
+  Alternatively, you can use the `ensurepip` module to install or upgrade Pip on Python 3:

```

Comment:
So does this not work on python2?  Should we leave the easy_install stuff in that case?

Reply:
Actually it's supported also for Python 2.7: https://docs.python.org/2.7/library/ensurepip.html
I'll change the line